engine/render/src/render/render_script.cpp

```diff
@@ -1180,9 +1180,41 @@ namespace dmRender
         return asset_handle;
     }
 
+    static dmGraphics::HRenderTarget CheckRenderTarget(lua_State* L, int index, RenderScriptInstance* i)
+    {
+        if(lua_isnumber(L, index))
+        {
+            return (dmGraphics::HRenderTarget) CheckAssetHandle(L, index, i->m_RenderContext->m_GraphicsContext, dmGraphics::ASSET_TYPE_RENDER_TARGET);
+        }
+        else if (dmScript::IsHash(L, index) || lua_isstring(L, index))
+        {
+            dmhash_t rt_id                  = dmScript::CheckHashOrString(L, index);
+            RenderResource* render_resource = i->m_RenderResources.Get(rt_id);
+
+            if (render_resource == 0x0)
+            {
+                char str[128];
+                char buffer[256];
+                dmSnPrintf(buffer, sizeof(buffer), "Could not find render target '%s' %llu",
+                    dmScript::GetStringFromHashOrString(L, index, str, sizeof(str)), (unsigned long long) rt_id); // since lua doesn't support proper format arguments
```

Comment:
If you use dmHashReverseSafe64(), you'll get the string if it exists, or `<unknown:1234567>` if it doesn't.

I think it's good to consolidate our printouts so that they get the same formatting. Using hash reverse is one step I believe.